src/worker.js

```diff
@@ -131,14 +143,20 @@ this.onmessage = function(e) {
         URL.revokeObjectURL(objectUrl);
       }
 #if MODULARIZE && !MODULARIZE_INSTANCE
+#if MINIMAL_RUNTIME
+      Module = {{{ EXPORT_NAME }}}(imports);
+#else
       Module = {{{ EXPORT_NAME }}}(Module);
 #endif
-      PThread = Module['PThread'];
```

Comment:
isn't it shorter to assign to `PThread` once instead of doing `Module['PThread']` many times as you changed it to?

Reply:
It is, but I wanted to avoid the number of global variables that would collide in meaning: having PThread and HEAPU32 here, but then loading the global JS script with the same vars can be confusing.

Proper code size optimizations will follow with the refactor to remove all of these Module['x'] exports later by moving the contents of src/worker.js into the main JS file (and keeping src/worker.js a minimal stub that only loads the main JS) - that will allow closure minification of all of this.